stock_accrual_report/wizard/wizard.py

```diff
@@ -59,54 +59,56 @@
 
 QUERY_LINE_MOVE = '''
 SELECT DISTINCT ol.id
-FROM {ttype}_order_line AS ol
+FROM %(ttype)s_order_line AS ol
 INNER JOIN
-    stock_move AS sm ON sm.{ttype}_line_id = ol.id
+    stock_move AS sm ON sm.%(ttype)s_line_id = ol.id
 WHERE
     sm.state IN ('done')
-    AND (sm.date BETWEEN '{date_start}' AND '{date_stop}')
-    AND sm.company_id = {company_id}
+    AND (sm.date BETWEEN %(date_start)s AND %(date_stop)s)
+    AND sm.company_id = %(company_id)s
 '''
 
 QUERY_LINE_INVOICE = '''
 SELECT DISTINCT ol.id
-FROM {ttype}_order_line AS ol
+FROM %(ttype)s_order_line AS ol
 INNER JOIN
-    {ttype}_order_line_invoice_rel AS olir ON olir.order_line_id = ol.order_id
+    %(ttype)s_order_line_invoice_rel AS olir
+    ON olir.order_line_id = ol.order_id
 INNER JOIN
     account_invoice_line AS ail ON ail.id = olir.invoice_id
 INNER JOIN
     account_invoice AS ai ON ai.id = ail.invoice_id
 WHERE
     ai.state IN ('open', 'paid')
-    AND (ai.date_invoice BETWEEN '{date_start}' AND '{date_stop}')
-    AND ai.company_id = {company_id}
+    AND (ai.date_invoice BETWEEN %(date_start)s AND %(date_stop)s)
+    AND ai.company_id = %(company_id)s
 '''
 
 QUERY_MOVE = '''
 SELECT DISTINCT order_id
-FROM {ttype}_order_line AS ol
+FROM %(ttype)s_order_line AS ol
 INNER JOIN
-    stock_move AS sm ON sm.{ttype}_line_id = ol.id
+    stock_move AS sm ON sm.%(ttype)s_line_id = ol.id
 WHERE
     sm.state IN ('done')
-    AND (sm.date BETWEEN '{date_start}' AND '{date_stop}')
-    AND sm.company_id = {company_id}
+    AND (sm.date BETWEEN %(date_start)s AND %(date_stop)s)
+    AND sm.company_id = %(company_id)s
 '''
 
 QUERY_INVOICE = '''
 SELECT DISTINCT order_id
-FROM {ttype}_order_line AS ol
```

Comment:
The section of `FROM %s` and `INNER JOIN %s` is not supported by `cr.execute(args, values)`